Doc/library/ctypes.rst

```diff
@@ -1408,6 +1408,10 @@ accessing the function through an attribute caches the result and therefore
 accessing it repeatedly returns the same object each time.  On the other hand,
 accessing it through an index returns a new object each time:
 
+.. doctest::
```

Comment:
Well, I didn't mentioned it because I assumed you are going to change "[...] each time:" to "[...] each time::" :)